crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -473,13 +476,13 @@ impl Call {
 	}
 }
 
-// Sign and submit an extrinsic using wallet integration.
-async fn submit_extrinsic_with_wallet(
+// Sign and submit an extrinsic using wallet integration, then returns the resulting events.
+pub(crate) async fn submit_extrinsic_with_wallet(
```

Comment:
This function is now reused in `pop up`, so it might make sense to move it to a common file. But I wasn’t sure about making additional changes.

Reply:
Agree! Moved in https://github.com/r0gue-io/pop-cli/pull/410/commits/7d87f17191e66f7d9dbfa689df7749feefff47de